pepper-apis/api-build.sh

```diff
@@ -293,6 +293,7 @@ DOCS_PROXIED_HOST=$DOCS_PROXIED_HOST
 VERSION=$1
 ENVIRONMENT=$2
 DIR=$3
+APPLICATION=unknown
```

Comment:
What is the expected behavior if the application is not passed in the arguments?

Reply:
This `APPLICATION` value filters its way to template generation.
Up to recently housekeeping and pepper-backend shared exact application.conf template .
Had need to generate slightly different .conf files for each.
So if APPLICATION=housekeeping , there is a conditional statement in a template that renders a param value in application.conf . If it is not housekeeping, then it render the default value for param.

I had made the param optional in the interest of keeping everything backwards compatible (i.e., previous usage was preserved). I introduced bug misjudging how GO templates would handle missing value.

Reply:
Thank you!